android/library/src/main/java/com/liferay/mobile/screens/webcontent/display/view/WebContentDisplayViewModel.java

```diff
@@ -28,4 +28,11 @@
 	 * @param html {@link WebContent} content
 	 */
 	void showFinishOperation(WebContent html);
+
+	/**
+	 * Customize the {@link WebContent} html with custom CSS.
+	 *
+	 * @return css style
+	 */
+	String getCustomCssStyle();
```

Comment:
the view model should not include this method. This is only intended to be overwritten in children views. Attributes in viewModel are used to communicate the view and the screenlet, and you don't want this method to be called from the screenlet, right?